### PR TITLE
added prepublish hook to have a concatenated version after install

### PR DESCRIPTION
**Pull request by quarkus**
just added a prepublish hook to have a local concatenated / minified version available after install

### CODE REVIEW

**Comment by dmlap** [comment]
Good thinking! I have no idea why travis is trying to run prepublish as part of the CI build, though.

**Comment by gkatsev** [comment]
I believe it's a bug with npm. We probably want to update our travis config to use a newer version of npm as well as using newer versions of node/iojs.

**Comment by gkatsev** [comment]
A workaround is to change the `"before_script"` property in `.travis.yml` to be `install` and append `npm install` to it.
```yaml
before_script:
- npm install -g grunt-cli
```
should turn into:
```yaml
install:
- npm install -g grunt-cli && npm install
```
See my changes here: https://github.com/videojs/videojs-contrib-ads/pull/72

**Comment by gkatsev** [comment]
@quarkus hey, want to make the change to the `.travis.yml` file?

**Comment by quarkus** [comment]
@gkatsev sorry .. i have been pretty busy over the last days. is the change correct ? i am not totally sure what i´m doing ere ;)

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
 - '0.10'
-before_script:
-- npm install -g grunt-cli
+install:
+- npm install -g grunt-cli && npm install
```
Comment (gkatsev) [inline .travis.yml]: perfect.

**Comment by gkatsev** [comment]
LGTM

**Comment by gkatsev** [comment]
Also, I just realized there's a better way of doing it. Instead of changing the `install` script in the `travis.yml`, we can just declare `grunt-cli` as a devDependency.

**Comment by dmlap** [comment]
@quarkus just tried this out with the 0.13 release and it worked like a charm. Thank you!

**Pull request merged**